tools/StaticAnalysis/Exceptions/Az.Network/SignatureIssues.csv

```diff
@@ -293,3 +293,6 @@
 "Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.UpdateAzureRmExpressRouteGatewayCommand","Set-AzExpressRouteGateway","1","8410","Parameter MaxScaleUnits of cmdlet Set-AzExpressRouteGateway does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
 "Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NewNetworkWatcherNetworkConfigurationDiagnosticProfileCommand","New-AzNetworkWatcherNetworkConfigurationDiagnosticProfile","1","8100","New-AzNetworkWatcherNetworkConfigurationDiagnosticProfile Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.InvokeNetworkWatcherNetworkConfigurationDiagnosticCommand","Invoke-AzNetworkWatcherNetworkConfigurationDiagnostic","1","8100","Invoke-AzNetworkWatcherNetworkConfigurationDiagnostic Does not support ShouldProcess but the cmdlet verb Invoke indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.GetAzureApplicationGatewayAvailableRequestHeaders","Get-AzApplicationGatewayAvailableRequestHeaders","1","8400","Get-AzApplicationGatewayAvailableRequestHeaders uses the noun 'AzApplicationGatewayAvailableRequestHeaders', which does not follow the enforced naming convention of using a singular noun for a cmdlet name.","Consider using a singular noun for the cmdlet name."
```

Comment:
@nirkum18 you can revert the changes made in this file now that this won't be flagged

Reply:
@cormacpayne removed these changes